loader/loader.go

```diff
@@ -300,6 +301,51 @@ func parseYAML(decoder *yaml.Decoder) (map[string]interface{}, PostProcessor, er
 	return converted.(map[string]interface{}), &processor, nil
 }
 
+// LoadConfigFiles ingests config files with ResourceLoader and returns config details with paths to local copies
+func LoadConfigFiles(ctx context.Context, configFiles []string, workingDir string, options ...func(*Options)) (*types.ConfigDetails, error) {
+	if len(configFiles) < 1 {
+		return &types.ConfigDetails{}, fmt.Errorf("no configuration file provided: %w", errdefs.ErrNotFound)
+	}
+
+	opts := &Options{}
+	config := &types.ConfigDetails{
+		ConfigFiles: make([]types.ConfigFile, len(configFiles)),
+		WorkingDir:  workingDir,
+	}
+
+	for _, op := range options {
+		op(opts)
+	}
+	opts.ResourceLoaders = append(opts.ResourceLoaders, localResourceLoader{})
+
+	for i, p := range configFiles {
+		for _, loader := range opts.ResourceLoaders {
+			_, isLocalResourceLoader := loader.(localResourceLoader)
```

Comment:
if local loader reject `Accept` it's highly probable it will also fail on `Load`, so why not just let it reject path and get this function return an error if no loader accepted the path ?

Reply:
Yes but you won't have the reason why, what I could do is changing the `Accept` signature to return an error in addition of the boolean, WDYT?

Reply:
the other option could be for `localResource Loader` to always accept path. The `Load` can return file not found error

Reply:
Ok I'll adapt the `Accept` function of resource loader